src/main/kotlin/uk/gov/dwp/dataworks/aws/AwsCommunicator.kt

```diff
@@ -540,4 +549,23 @@ class AwsCommunicator {
             return false
         }
     }
+
+
+    fun rdsExecuteStatement(sql: String, parameters: Collection<SqlParameter>? = null): ExecuteStatementResponse {
+        try{
+            val executeStatementRequest = ExecuteStatementRequest
+                .builder()
+                .secretArn(configurationResolver.getStringConfig(ConfigKey.RDS_CREDENTIALS_SECRET_ARN))
+                .resourceArn(configurationResolver.getStringConfig(ConfigKey.RDS_CLUSTER_ARN))
+                .database(configurationResolver.getStringConfig(ConfigKey.RDS_DATABASE_NAME))
+                .parameters(parameters)
+                .sql(sql)
+
+            return awsClients.rdsDataClient.executeStatement(executeStatementRequest.build())
```

Comment:
Is there a reason that we are not closing the `builder` in the request?

Reply:
There used to be a if statement that added the `.parameters()` to the request if not null, then realised I can just pass the possibly null value to the builder anyway, but forgot to change where the `build()` gets called. Doesn't really make any difference